charts/data-prepper/Chart.yaml

```diff
@@ -15,13 +15,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.1.2
```

Comment:
I somewhat arbitrarily picked v0.1.2 here since I have another PR open where I picked 0.1.1.

What is the recommendation for picking versions here?